script/adapters/Axelar.s.sol

```diff
@@ -8,21 +8,12 @@ import {FullDeployer} from "script/FullDeployer.s.sol";
 
 // Script to deploy CP and CP with an AxelarScript Adapter.
 contract AxelarDeployer is FullDeployer {
-    function run() public {
-        uint16 localCentrifugeId = uint16(vm.envUint("CENTRIFUGE_ID"));
+    function deploy() public {
```

Comment:
CR. I think we can not inherit from `FullDeployer`.

`wire` is assuming someone has initializate the contracts, but from this script perspective, everything is 0. 

I think we need code wire here, or pass the correct `gateway` and `root` addresses by env